pep-0567.rst

```diff
@@ -551,6 +604,53 @@ code unmodified, but will automatically enable support for
 asynchronous code.
 
 
+Examples
+========
+
+Converting code that uses threading.local()
+-------------------------------------------
+
+A typical code that uses ``threading.local()`` usually looks like
```

Comment:
code -> code fragment (code is not countable so you can't say "a code").

Reply:
fixed